src/SourceLocator/Type/Composer/Psr/Psr4Mapping.php

```diff
@@ -68,7 +68,7 @@ public function resolvePossibleFilePaths(Identifier $identifier): array
                     return [];
                 }
 
-                return array_map(static fn (string $path): string => rtrim($path, '/') . '/' . $subPath . '.php', $paths);
+                return array_map(static fn (string $path): string => $path . '/' . $subPath . '.php', $paths);
```

Comment:
The ending `/` in the path is valid in PSR-0 and PSR-4 mappings - should probably be tested.

Reply:
It's already removed in https://github.com/Roave/BetterReflection/blob/5.1.x/src/SourceLocator/Type/Composer/Psr/Psr4Mapping.php#L44 and tested as well.